resources/home/dnanexus/generate_workbook/utils/vcf.py

```
@@ -1064,19 +1042,24 @@
         elif row.get('hgvsp'):
-            text += f"HGVSp: {add_none(row.get('hgvsp', ''))}\n"
+            text += f"HGVSp: {add_none(row.get('hgvsp'))}\n"
 
         if row.get('existing_variation', '').replace('.', ''):
             text += f"dbSNP: {row.get('existing_variation', '')}\n"
 
-        text += f"Allele Frequency (VAF): {add_none(str(row.get('af', '')))}"
+        if row.get('af') == '.':
+            value = 'None'
+        else:
+            value = f"{float(row.get('af')):.1%}"
```

Ruff (0.11.9)

Use ternary operator `value = 'None' if row.get('af') == '.' else f"{float(row.get('af')):.1%}"` instead of `if`-`else`-block

Replace `if`-`else`-block with `value = 'None' if row.get('af') == '.' else f"{float(row.get('af')):.1%}"`

(SIM108)